lib/core-integration/src/Test/Integration/Scenario/API/Network.hs

```diff
@@ -32,3 +40,11 @@ spec = do
         eventually $ do
             r <- request @ApiNetworkInformation ctx endpoint Default Empty
             verify r [ expectFieldEqual syncProgress Ready ]
+
+    describe "NETWORK - v2/network/information - Methods Not Allowed" $ do
+        let matrix = ["POST", "CONNECT", "TRACE", "OPTIONS"]
```

Comment:
This is really just testing servant, not our application, isn't it?

Reply:
To some extend yes, it verifies that our setup matches our expectations. A blackbox test if you prefer.

Reply:
maybe this blog post is of some interest to us : https://blog.ploeh.dk/2019/09/23/unit-testing-wai-applications/